lib/mayaUsd/ufe/UsdUndoRenameCommand.cpp

```diff
@@ -214,7 +216,7 @@ void UsdUndoRenameCommand::renameRedo()
     }
 
     // send notification to update UFE data model
-    sendNotificationToAllStageProxies(_stage, prim, srcPath, dstPath);
+    sendNotificationToAllStageProxies(_stage, _ufeDstItem->prim(), srcPath, dstPath);
```

Comment:
Make sure to use the new prim when sending notifs (not the source prim from before rename).